mobile/src/bundles/employer/validation-schemas/employer-onboarding-form.validation-schema.ts

```diff
@@ -11,8 +11,8 @@ const EmployerOnboardingFormValidationSchema = joi.object<
     EmployerOnboardingFormDto,
     true
 >({
-    profilePhoto: joi.object().required(),
-    companyLogo: joi.object().required(),
+    profilePhoto: joi.object().allow(null),
+    companyLogo: joi.object().allow(null),
     fullName: joi
```

Comment:
https://docs.google.com/spreadsheets/d/1OM1GunJPbMa6u0Dn4rm9Cgh32PBfMNOxv588UZYBM_4/edit#gid=0 - Here are the validation rules.
If it is not bold, it is not required, I think.

Reply:
I asked QA team for employer onboarding form. Can you also ask them about your page? 
Just so that later we don't have tasks with bugs about non-compliance

Reply:
I have asked QA. I will change it some fields to required.